azkaban-web-server/src/main/java/azkaban/webapp/AzkabanWebServer.java

```diff
@@ -1261,6 +1205,13 @@ public void close() {
     }
     scheduleManager.shutdown();
     executorManager.shutdown();
+    try {
+      server.stop();
+    } catch (Throwable t) {
```

Comment:
`Throwable` is the mother class of all `catch`able classes in java. `Exception` is a subclass of Throwable. The idea is to catch all errors at this point since we are trying to shutdown the server.